src/lib/ical/common.ts

```diff
@@ -27,7 +27,7 @@ export interface IcalOutput {
   [key: string]: string | string[] | IcalOutput[];
 }
 
-function cleanIcalKey(key: string): string {
+export function cleanIcalKey(key: string): string {
```

Comment:
Only export functions that are used outside of this file.

Reply:
Resolved by https://github.com/EthanThatOneKid/acmcsuf.com/pull/337/commits/8a132fa3f2a555bdb5716aa3400373d41bdffef3.